packages/deno/src/index.ts

```diff
@@ -74,6 +74,11 @@ export {
   startInactiveSpan,
   startSpanManual,
   metrics,
+  inboundFiltersIntegration,
+  linkedErrorsIntegration,
+  moduleMetadataIntegration,
```

Comment:
Do we want to export this * `requestDataIntegration` here? Previously, these were not accessible directly from `@sentry/deno`.